includes/embeds/class-amp-core-block-handler.php

```diff
@@ -421,6 +445,12 @@ static function ( $matches ) use ( $modal_state_property ) {
 		// Delete other micromodal-related data attributes.
 		$block_content = preg_replace( '/\sdata-micromodal-close/', '', $block_content );
 
+		// Delete `data-wp-*` attributes.
+		if ( $is_interactive_block ) {
+			$block_content = preg_replace( '/\sdata-wp-[^=]+="[^"]*"/', '', $block_content );
+			$block_content = preg_replace( '/\sdata-wp-[^=]+=\'[^\']*\'/', '', $block_content );
+		}
+
```

Comment:
These can remain, right? They just won't do anything.

Reply:
Yes, they won't, apart from increasing content length by a few bytes 😅

Reply:
Just that their removal could cause problems due to the string replacement not using parsed HTML, and they take additional (inconsequential) server time to remove. IMO, I think we can just leave them.